index.js

```diff
@@ -42,6 +46,7 @@ module.exports = {
     "max-line-length": [ 80, {
       ignore: "non-comments",
     } ],
+    "max-nesting-depth": 3,
```

Comment:
Hmm, I don't think counting that way is correct because the `1` is not nesting, that's just a normal ruleset. It makes sense to count `1` from the first nested ruleset and it seems that's what stylelint does too.